src/policy_downloader.rs

```diff
@@ -55,7 +63,6 @@ impl Downloader {
         &mut self,
         policies: &mut HashMap<String, Policy>,
         destination: &str,
-        verification_config: &LatestVerificationConfig,
```

Comment:
The main issue with that is that there are combinations that don't work together: build the downloader without verify and call to `download_policies` with `verification_config` and the other way around. The downloader once built, either is able to download verifying, or not. Is not a trait of the function call exclusively, but a trait of how you build the downloader initially.

The boolean as far as I understand is only to optimize the unit tests so that only one Downloader is built (and TUF data downloaded only once), when `verification_config` can be called several times on that. I would not allow to call degenerated cases for increasing the testing time (if this is the case).

Reply:
Okay, yes. I see your point. Let me update the PR.

Reply:
Done.